docs/AboutUs.adoc

```diff
@@ -17,4 +17,11 @@ image::limerencee.png[width="150", align="left"]
 Role: Project Manager/Developer +
 Responsibilities: Deliverables and Deadlines
 
+=== Lye Jian Wen
+image::lye-jw.png[width="150", align="left"]
+link:https://https://github.com/lye-jw[[github]] [<<lye-jw#, portfolio>>]
```

Comment:
Your GitHub link is broken because there is 2 `https://` in your GitHub link

Reply:
Noted, thanks!